test/integration/tags/inline_comment_test.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class InlineCommentTest < Minitest::Test
+  include Liquid
+
+  def test_tag_in_different_styles
+    assert_template_result('', '{% # This text gets ignored %}')
+    assert_template_result('', '{%# This text gets ignored #%}')
+    assert_template_result('', '{%# This text gets ignored %}')
+    assert_template_result('', '{%#- This text gets ignored -#%}')
+  end
+
+  def test_test_syntax_error
+    assert_template_result('fail', '{% #This doesnt work %}')
+
+    assert false 
+  rescue 
+    # ok good
+  end
```

Comment:
If I understand correctly, this test is saying that a space is required after `#`.